src/main/java/org/radarbase/output/source/StorageIndex.kt

```diff
@@ -0,0 +1,13 @@
+package org.radarbase.output.source
+
+import java.nio.file.Paths
+
+interface StorageIndex {
```

Comment:
It might be good to add a line of documentation to these interfaces (i.e. StorageIndex, StorageNode, StorageIndexManager)